app/views/houston/dashboards/staging/_up_next.html.erb

```diff
@@ -7,10 +7,13 @@
             <%= project_data[:project].slug %>
           </span>
         </td>
-        <td class="staging-pr-title">
+        <td class="up-next-title">
           <%= link_to truncate(pull_request.title, length: 100), pull_request.html_url, target: "_blank" %>
         </td>
-        <td class="staging-user">
+        <td class="checkboxes">
+          <%= raw(checkboxes(pull_request)) %>
```

Comment:
Instead of using `raw` in a template, it's a better idea to have your helper (that constructs HTML) mark that HTML as safe like so:

```diff
  def checkboxes(pull_request)
    checked = completed_checkboxes(pull_request)
    total = total_checkboxes(pull_request)
-   "<span class='label'>#{checked}/#{total}</span>" unless total < 1
+   "<span class='label'>#{checked}/#{total}</span>".html_safe unless total < 1
  end
```